File: 09/Q3/guess.js
// npm install prompt-sync
const prompt = require('prompt-sync')({sigint: true});

let n = Math.floor(Math.random() * 100) + 1

// Math.random() -> 0..0.999999
// Math.random() * 100 -> 0 .. 99.999999
while (true) {
    let guess = parseInt(prompt("Guess a number from 1 to 100"))
    if (guess > n) {
        console.log("Try lesser number")
    }
    else if (guess < n) {
        console.log("Try bigger number")
    }
    else {
        console.log("Whooraay! You won!")
        break
    }
}

File: past/09/Q3/guess.java
import java.util.Random;
import java.util.Scanner; 

public class Main{
  
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);   

        Random rand = new Random();
        int secret = rand.nextInt(100) + 1;
        int guess = -1;

        while (true) {
            System.out.println("Guess a number from 1 to 100");
            guess = sc.nextInt();

            if (guess > secret) {
                System.out.println("Try lesser number");
            }
            else if (guess < secret) {
                System.out.println("Try bigger number");
            }
            else {
                System.out.println("Whooraay! You won!");
            }
        }
    }
}